readme.md

```diff
@@ -99,6 +99,27 @@ const result = await pRetry(run, {
 
 If the `onFailedAttempt` function throws, all retries will be aborted and the original promise will reject with the thrown error.
 
+##### signal
+
+Type: `AbortSignal`
+
+An AbortSignal object. Useful to remotely abort an ongoing operation.
```

Comment:
Use a description similar to this one: https://github.com/sindresorhus/execa#signal-1 Including the Node.js requirement text.